backend/game_engine/common.py

```diff
@@ -211,5 +224,64 @@ def to_json(self):
         return json.dumps(self.to_dict())
 
 
-def log_message(message):
-    logging.info(message)
+def log_message(message, game_attributes):
+    #game_attributes is usually self
+    print("LOG_MESSAGE CALLED --------------------")
+    phone_numbers=[]
+    tiktok=""
+    team_id=""
+    tribe_id=""
+    game_id=""
+    player_id=""
+    if game_attributes:
```

Comment:
By using a dict as the type for additional tags, we can simplify this code by iterating over all key value pairs in the additional tags dict and calling scope.set_tag accordingly. This puts more work on the caller to supply the tags that they want to be logged, but I think this is a good thing.